pkg/fleets/controller.go

```diff
@@ -410,27 +411,37 @@ func (c *Controller) rollingUpdateDeployment(fleet *agonesv1.Fleet, active *agon
 	if err != nil {
 		return 0, err
 	}
-	if err := c.rollingUpdateRest(fleet, rest); err != nil {
+	if err := c.rollingUpdateRest(fleet, active, rest); err != nil {
 		return 0, err
 	}
 	return replicas, nil
 }
 
+func getReadyReplicaCountForGameServerSets(gss []*agonesv1.GameServerSet) int32 {
```

Comment:
Should this be in fleet.go with SumStatusReplicas ? Somelike like `SumStatusReady`? (I'm actually a little surprised we don't have this already!)

Reply:
Yes, that would be a better place